resources/monitoring/.helmignore

```diff
@@ -19,3 +19,8 @@
 .project
 .idea/
 *.tmproj
+# helm/charts
+OWNERS
```

Comment:
Git should not ignore OWNERS file changes

Reply:
It's `helmignore`, but you are right, we don't need this change